app/assets/javascripts/password-strength.js

```diff
@@ -8,7 +8,7 @@ var PasswordStrengthCalculator = function(inputId, strengthLabelId) {
   //var specialchars = new RegExp('([!,%,&,@,#,$,^,*,?,_,~])');
 
   var getPasswordStrength = function(value){
-    if (value.length > 8) { characters = 1; } else { characters = 0; };
+    if (value.length >= 8) { characters = 1; } else { characters = 0; };
```

Comment:
Magic numbers should be constants :crying_cat_face: 
What is 8? How strong is 4?